src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -636,6 +637,7 @@ const generateFrameworkDescription = (frameworkName, virtualCluster, config, raw
       labels: {
         userName: jobInfo.userName,
         virtualCluster: jobInfo.virtualCluster,
+        logPathInfix: jobInfo.logPathInfix,
```

Comment:
Can put this into annotations? Maybe we don't need to filter framework according to `logPathInfix`